i18n/tr/cachyos_hello.ftl

```diff
@@ -0,0 +1,82 @@
+# About dialog
+about-dialog-title = CachyOS Hello
+about-dialog-comments = CachyOS Hoşgeldin Ekranı
+
+# Tweaks page
+tweaks = İyileştirmeler
+fixes = Hata düzeltmeler
+applications = Uygulamalar
+removed-db-lock = Pacman veri tabanı kilidi kaldırıldı!
+lock-doesnt-exist = Pacman veri tabanı kilidi bulunmamaktadır!
+orphans-not-found = Hiçbir öksüz paket bulunamadı!
+package-not-installed = '{$package_name}' paketi kurulmamıştır!
+
+# Application Browser page
+advanced-btn = gelişmiş
+reset-btn = sıfırla
+update-system-app-btn = SİSTEMİ GÜNCELLE
+application-column = Uygulama
+description-column = Açıklama
+install-remove-column = Kur/Kaldır
+advanced-btn-tooltip = Genişletilmiş paket seçeneklerini aç veya kapa
+reset-btn-tooltip = Mevcut seçimleri eski haline getir…
```

Comment:
instead of "eski haline getir", "sıfırla" is more suitable and also more concise.